integration-cli/docker_cli_rm_test.go

```diff
@@ -19,6 +20,9 @@ func (s *DockerSuite) TestRmContainerWithRemovedVolume(c *check.C) {
 	}
 	defer os.RemoveAll(tempDir)
 
+	out, _, _ := dockerCmdWithStdoutStderr(c, "ps", "-a")
+	fmt.Printf("out: %s", out)
```

Comment:
Before resorting to Printf, I did try c.Log(). However, I failed to see messages on stdout/stderr. Both Log and Logf write to a logb (logbuffer) or if a io.Writer has been initialized, then optionally write to it. RunConf has an io.Writer which if not initialized, is set to os.Stdout. So technically, it should work. Maybe we are overriding some config along the way.

In any case, printf works. Plus, it is added only temporarily to debug unclean-container-state. That's it.